frontend/.eslintrc.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     // Sometime we need to specify any type, so it's rather useful
     "@typescript-eslint/no-explicit-any": 0,
 
+    "react/jsx-closing-bracket-location": 0,
```

Comment:
Why?

Reply:
because construction
<Comp
    attr1=..
    attr2=..
>
  .. code here
</Comp>

looks ugly, what is the point of moving > at the separate line? it's not improve readability to me. does it do for you? it just looks like nonsense to me. why should we keep this rule?

Reply:
I don't think that it is ugly as it helps you to quickly separate `attrs` from `code here`, but if it really important for you I am ok with it